go.mod

```diff
@@ -15,9 +15,9 @@ require (
 	github.com/gofrs/uuid v4.4.0+incompatible
 	github.com/golang-jwt/jwt/v4 v4.5.0
 	github.com/google/go-cmp v0.5.9
-	github.com/google/uuid v1.3.1
+	github.com/google/uuid v1.4.0
 	github.com/hashicorp/go-azure-helpers v0.64.0
-	github.com/hashicorp/go-azure-sdk v0.20231207.1122031
+	github.com/hashicorp/go-azure-sdk v0.20231211.1135715
```

Comment:
FYI this has been pulled out into https://github.com/hashicorp/terraform-provider-azurerm/pull/24217

Reply:
I'll rebase once I have something working. :)